tests/FakeItEasy.Tests/Core/DefaultFakeManagerAccessorTests.cs

```diff
@@ -115,7 +115,7 @@ public void Should_fail_when_getting_manager_from_object_where_a_wrong_typed_fak
             // Assert
             exception.Should()
                 .BeAnExceptionOfType<ArgumentException>()
-                .WithMessage("The specified object is not recognized as a fake object.");
+                .WithMessage($"Object '{proxy.ToString()}' of type '{proxy.GetType().FullNameCSharp()}' is not recognized as a fake object.");

```

Comment:
Uh... no, I did it on line 118 (you can see it on the "conversation" page). Github reviews are riddled with bugs 😕